README.md

```diff
@@ -20,6 +20,10 @@ This module allows you to seamlessly integrate [the MyParcel services] into your
 
 For the manual and more information, check out our [PrestaShop module guide] on the [MyParcel Developer Portal].
 
+### Upgrading to v4.x
+
+When upgrading to v4.x it is necessary to clear the cache of Prestashop. To do so, go to `Advanced Parameters` -> `Performance` and click on `Clear cache`.
```

Comment:
Zou hier van maken it _may_ be necessary omdat het alleen van toepassing is als je de cache aan hebt staan (en zelfs dan misschien niet altijd?)

```suggestion
When upgrading to v4.x it may be necessary to first clear the cache of Prestashop. To do so, go to `Advanced Parameters` -> `Performance` and click on `Clear cache`, then upload the plugin as normal.
```